pyramid_zipkin/tween.py

```diff
@@ -60,6 +74,10 @@ def _get_settings_from_request(request):
             " and a message as params and logs the message via scribe/kafka."
         )
 
+    context_stack = _get_dotted(request, settings.get('zipkin.request_context'))
+    if not context_stack:
```

Comment:
We might want a stricter check here (`if context_stack is None`), as we don't want to rely on the fact that the object evaluates to `True` even when the stack is empty.